styles/colors-gallium-day.less

```diff
@@ -11,20 +11,22 @@
 @agate-highlight-h: var(--agate-highlight-h, 246);
 @agate-highlight-s: var(--agate-highlight-s, 25%);
 @agate-highlight-l: var(--agate-highlight-l, 100%);
-@agate-background: @agate-white;
-@agate-foreground: black;
-@agate-background-active: fade(black, 15%);
-@agate-foreground-active: @agate-white;
-@agate-background-focus: @agate-accent;
-@agate-foreground-focus: @agate-white;
-@agate-gradient: linear-gradient(to bottom, ~"hsla(var(--agate-accent-h, 0), var(--agate-accent-s, 0), var(--agate-accent-l, 0), 0.5)", @agate-accent);
-@agate-black:       #131313;
+@agate-black:       #000000;
+@agate-white:       #ffffff;
 @agate-darker-gray: #373737;
 @agate-dark-gray:   #5e5e5e;
 @agate-gray:        #8a8a8a;
 @agate-medium-gray: #c9c9c9;
 @agate-light-gray:  #ededed;
-@agate-white:       white;
+@agate-seal-brown-gray: #131313;
+@agate-night-rider-gray: #333333;
```

Comment:
I think these two will make more sense as `@gallium-seal-brown-gray` and `@gallium-night-rider-gray`, since they aren't common to all skins (only things that are common to all skins should have the `@agate` prefix.

Also, these names (even though they're weird ;) ) do represent specific finite color values, rather than generalized terms that a skin could define for itself. e.g. it would be unexpected to have `@gallium-seal-brown-gray` represent anything other than `#131313`. Black and white fit into this category too, but they're so common it's hard to _not_ include them.